src/components/InstructorRevenueCard/index.js

```diff
@@ -23,95 +28,81 @@ export default ({revenueUrl}) => (
           return null
         }
 
-        const currentMonthDates = map(data, month => month.month)
+        const currentMonthNames = map(data, month => prettyMonthName(month.month))
         const currentRevenuePoints = map(data, month => month.revenue)
         const currentMinutesPoints = map(data, month => month.minutes_watched)
-        const revenueColor = '#4786ff' // # blue
-        const minutesColor = '#171e27' // # base-secondary
 
         return (
           <OpenToggle>
             {({isOpen, handleOpenToggleClick}) => (
               <Card>
                 <div className='flex-l justify-between-l'>
 
-                  <Maybe condition={!isOpen}>
-                    <div className='pa5 nowrap-l'>
-
-                        <div className='mb4'>
-                          <RevenuePeriod
-                            title={<Text message='instructorRevenue.currentMonth.title' />}
-                            revenue={currentMonthRevenue.revenue}
-                            subscriberMinutes={currentMonthRevenue.minutes_watched}
-                          />
-                        </div>
-
-                        <RevenuePeriod
-                          title={<Text 
-                            message='instructorRevenue.previousMonths.title' 
-                            values={[currentTotalRevenue.monthCount]} 
-                          />}
-                          revenue={currentTotalRevenue.revenue}
-                          subscriberMinutes={currentTotalRevenue.minutes_watched}
-                        />
+                  <div 
+                    className='pa5 nowrap-l'
+                    style={{
+                      boxShadow: '10px 0px 10px -10px rgba(0, 0, 0, 0.1)',
+                    }}
```

Comment:
@vojtaholik Yeah no problem. This PR is already merged so I made a new story for some quick style tweaks that I'll get to later if you want to add to it; basically any small text/color/padding type things that can be done with CSS values are fair game for this story:
https://github.com/eggheadio/egghead-instructor-center/issues/216